config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  resources :users, shallow: true do
+    resources :comments
+    resources :posts
```

Comment:
- [ ] try to avoid unnecessarily exposing end-points

Reply:
No clue

Reply:
If I remove shallow nothing works, thanks.